src/Modules/Windows-Full/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="1DA87E53-152B-403E-98DC-74D7B4D63D59"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="� Microsoft Corporation. All rights reserved."
+Copyright="© Microsoft Corporation. All rights reserved."

```

Comment:
Instead of reverting the change, you added a BOM.

That's a reasonable change to make (we have another PR to clean these up), but to keep this PR clean, I'd prefer you undo the change and not change the encoding.